mkt/tags/models.py

```diff
@@ -37,25 +37,23 @@ def can_reverse(self):
     def get_url_path(self):
         return reverse('tags.detail', args=[self.tag_text])
 
-    def save_tag(self, addon):
+    def _get_m2m_name(self, obj):
+        model = obj._meta.model_name
+        if model == 'webapp':
+            return 'tags'
+        elif model == 'website':
+            return 'keywords'
```

Comment:
also, if you figure this out, you can use that to remove the `m2m_name` parameter that I added to `attach_tags()` below?